dev-docs/show-native-ads.md

```diff
@@ -100,42 +104,48 @@ Each key's value is an object with several fields.  Most important is the `requi
   </p>
 </div>
 
+<a name="native-object" />
+
 {% highlight js %}
+
 pbjs.addAdUnits({
-  code: slot.code,
-  sizes: slot.size,
-  mediaType: "native",
-  nativeParams: {
-    title: {
-      required: true,
-      len: 80
-    },
-    body: {
-      required: true
-    },
-    brand: {
-      required: true
-    },
-    image: {
-      required: true
-    },
-    clickUrl: {
-      required: true
-    },
-  },
-  bids: [
-    {
-      bidder: 'appnexusAst',
-      params: {
-        placementId: '9880618'
-      }
-    },
-  ]
+    code: slot.code,
+    sizes: slot.size,
+    mediaTypes: {
+        native: {
+            image: {
+                required: true
+            },
+            title: {
+                required: true,
+                len: 80
+            },
+            sponsoredBy: {
+                required: true
+            },
+            clickUrl: {
+                required: true
+            },
+            body: {
+                required: true
+            },
+            icon: {
+                required: true
+            },
+        },
+        bids: [{
+            bidder: 'appnexusAst',
+            params: {
+                placementId: '9880618'
+            }
+        }, ]
+    }
 });
+
 {% endhighlight %}
 
 {: .alert.alert-danger :}
-For each native ad unit, all of the bidders within that ad unit must have declared native support in supportedMediaTypes as defined in the adapter's call to registerBidAdapter().  If there are any bidders without native support in a native ad unit, the request won't be made.
+For each native ad unit, all of the bidders within that ad unit must have declared native support in their adapter.  If there are any bidders without native support in a native ad unit, the request won't be made.  For a list of bidders with native support, see [Bidders with Video and Native Demand]({{site.baseurl}}/bidders.html#bidders-with-video-and-native-demand).
```

Comment:
This doesn't seem to align with observed behavior (and I'm not sure that this should be the expected behavior)

Reply:
Did some reading of the code and confirmed that the expected behavior is for Prebid to filter out non-native-enabled bidders's bids from native auctions (https://github.com/prebid/Prebid.js/blob/master/src/prebid.js#L399)

Did some testing and am updating the note text to read:

<blockquote>
For each native ad unit, all of the bidders within that ad unit must have declared native support in their adapter if you want ads to appear.  If there are any bidders without native support in a native ad unit, requests will not be made to those bidders.
</blockquote>